src/uyuni_health_check/exporter/uyuni_health_exporter.py

```diff
@@ -186,9 +244,21 @@ def collect(self):
                 print(error)
 
     start_http_server(port)
+    tracemalloc.start()
+    snapshot = tracemalloc.take_snapshot()
     uyuni_data_gatherer = UyuniDataGatherer()
     REGISTRY.register(UyuniMetricsCollector(uyuni_data_gatherer))
     while True:
         # period between collection
         time.sleep(frequency)
         uyuni_data_gatherer.refresh()
+        gc.collect()
+        snapshot2 = tracemalloc.take_snapshot()
+        top_stats = snapshot2.compare_to(snapshot, "lineno")
+        print("[ Top 10 differences ]")
+        for stat in top_stats[:10]:
+            print(stat)
```

Comment:
Isn't that a profiling tooling reminder to remove?

Reply:
Correct. Give me a sec to clean that up :)